Source/WebKit/WebProcess/WebPage/WebPage.h

```diff
@@ -1274,6 +1275,12 @@ class WebPage final : public API::ObjectImpl<API::Object::Type::BundlePage>, pub
     OptionSet<WebCore::DragSourceAction> allowedDragSourceActions() const { return m_allowedDragSourceActions; }
 #endif
 
+#if ENABLE(MODEL_PROCESS)
+    void requestInteractiveModelElementAtPoint(WebCore::IntPoint clientPosition);
+    void stageModeSessionDidUpdate(std::optional<WebCore::ElementIdentifier>, WebCore::TransformationMatrix);
```

Comment:
```suggestion
    void stageModeSessionDidUpdate(std::optional<WebCore::ElementIdentifier>, const WebCore::TransformationMatrix&);
```

Nit - probably `const&` or `&&` here to prevent unnecessary copies of a (somewhat-large) set of values.